test/integration/default_managedclusterset_controller_test.go

```diff
@@ -60,4 +60,6 @@ var _ = ginkgo.Describe("DefaultManagedClusterSet", func() {
 		}, eventuallyTimeout, eventuallyInterval).ShouldNot(gomega.HaveOccurred())
 	})
 
+	// TODO:
```

Comment:
do you want to add test here?

Reply:
Actually i wrote some case to test whether the spec will be rollback is changed, but seems like we can't do this now. 
```
ManagedClusterSet.cluster.open-cluster-management.io "default" is invalid: spec.clusterSelector.selectorType: Unsupported value: "anything": supported values: "LegacyClusterSetLabel"
  occurred
```
So i just add the TODO comment here, warning that there is no integration test case for changing spec.